contracts/DEX.sol

```diff
@@ -9,6 +9,8 @@ contract DEX {
     event CDTToSSVConverted(uint256 amount);
     event SSVToCDTConverted(uint256 amount);
```

Comment:
I referred to these events in the previous PR and I didn't see a change.
It's very unclear what `amount` represents in each event.
The relevant wallet is not specified.
In addition, it can be inferred from other events within the transaction (Transfer).

Reply:
fixed